num_list.py

```diff
@@ -19,5 +20,8 @@ def print_each_number_and_its_square():
     # The square of 10 is 100
     # ...etc
     # TODO - Write your code below this line.  (Keep the indentation)
-    
+    nums = [12, 10, 32, 3, 66, 17, 42, 99, 20]
```

Comment:
`nums` has already been declared at the top of the file. There is no need to reset the variable here.